### PR TITLE
gengapic: add import comment in example snippets

### LINKED ISSUE #107: gapic: comment type imports beside example code snippets
_Copied from [gapic-generator #2620](https://github.com/googleapis/gapic-generator/issues/2620)_

> There has been feedback indicating confusion around the difference between the genproto pkg and the gapic pkg. Because of naming & alias name misunderstandings, some see the former as the "old" client and the latter as the "new" one, which then gets folks confused as to why they need both imported.
> 
> The genproto pkg contains the generated types for the API protos, while the gapic pkg contains the generated client that consumes them.
> 
> To help clear this up, the generated example snippets should include a comment above the use of the genproto types indicating where it comes from.

cc @broady @jadekler

### PR DESCRIPTION
**Pull request by noahdietz**
Fixes #107 
Fixes [google-cloud-go#1356](https://github.com/googleapis/google-cloud-go/issues/1356)

In generated example snippets, add comment beside request payloads indicating import path of struct and for the iterator type.

@broady @jadekler can you review?

### CODE REVIEW

**Comment by broady** [comment]
Sorry to be a pain, and I think this is harder to implement, is something like this easily doable?

```
func ExampleFooClient_BidiThings() {
	// import "mypackagepb" mypackage
	// import "google.golang.org/api/iterator"

	ctx := context.Background()
	c, err := Bar.NewFooClient(ctx)

// snip

}
```

**Comment by noahdietz** [comment]
PTAL

File: internal/gengapic/testdata/foo_example.want [hunk 1]
```diff
@@ -66,6 +70,9 @@ func ExampleFooClient_GetBigThing() {
 }
 
 func ExampleFooClient_GetManyThings() {
+	// import "mypackage" mypackagepb
```
Comment (broady) [inline internal/gengapic/testdata/foo_example.want]: other way around - quotes around the package import, the name its imported as is unquoted

Comment (noahdietz) [reply internal/gengapic/testdata/foo_example.want]: That is how this is implemented, the example is just a mediocre expression of it. `mypackagepb` is in fact the name while `"mypackage"` is the import path. :sweat_smile:

Comment (broady) [reply internal/gengapic/testdata/foo_example.want]: in that case, they need to be swapped!

Comment (broady) [reply internal/gengapic/testdata/foo_example.want]: (sorry, i see that i messed that up in my suggestion above. whoops!)

File: internal/gengapic/example.go [hunk 2]
```diff
@@ -71,6 +71,24 @@ func (g *generator) exampleMethod(pkgName, servName string, m *descriptor.Method
...
 
 	p("func Example%sClient_%s() {", servName, m.GetName())
+
+	pf, err := g.pagingField(m)
+	if err != nil {
+		return err
+	}
+
+	if *m.OutputType != emptyType {
+		p("// import \"%s\" %s", inSpec.Path, inSpec.Name)
```
Comment (broady) [inline internal/gengapic/example.go]:
```suggestion
		p(`// import %s "%s"`, inSpec.Name, inSpec.Path)
```

File: internal/gengapic/testdata/foo_example.want (hunk 1 above)
Comment (noahdietz) [reply internal/gengapic/testdata/foo_example.want]: Done

File: internal/gengapic/example.go (hunk 2 above)
Comment (noahdietz) [reply internal/gengapic/example.go]: Thanks! committed it with `.want` file changes

**Comment by broady** [comment]
LGTM++

**Pull request merged**